client/components/head/index.jsx

```diff
@@ -0,0 +1,127 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+
+const Head = ( { title = 'WordPress.com', faviconURL, children, cdn } ) => {
+	return (
+		<head>
+			<title>{ title }</title>
+
+			<meta charSet="utf-8" />
+			<meta httpEquiv="X-UA-Compatible" content="IE=Edge" />
+			<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
+			<meta name="format-detection" content="telephone=no" />
+			<meta name="mobile-web-app-capable" content="yes" />
+			<meta name="apple-mobile-web-app-capable" content="yes" />
+			<meta name="referrer" content="origin" />
+
+			<link
+				rel="shortcut icon"
+				type="image/vnd.microsoft.icon"
+				href={ faviconURL }
+				sizes="16x16 32x32"
+			/>
+			<link rel="shortcut icon" type="image/x-icon" href={ faviconURL } sizes="16x16 32x32" />
+			<link rel="icon" type="image/x-icon" href={ faviconURL } sizes="16x16 32x32" />
+
+			<link
```

Comment:
We could cut out ~100 lines with something like:
```jsx
[64, 96, 192, ...].map( px => (
  <link 
     rel="icon" 
     type="image/png"
     href={cdn + `/i/favicons/apple-touch-icon-${px}x${px}.png`}
))
```

Reply:
True, the first ones have a slightly different path but still that would save some lines. As this PR is already merged I'm going to address this in a follow-up PR if that's okay?